lib/cli.js

```diff
@@ -10,5 +10,11 @@ module.exports = CLI
   .option('-p, --patch', 'create a patch changelog')
   .option('-m, --minor', 'create a minor changelog')
   .option('-M, --major', 'create a major changelog')
+  .option('-x, --exclude', 'exclude selected commit types (comma separated)', list)
   .option('-f, --file [file]', 'file to write to, defaults to ./CHANGELOG.md, use - for stdout', './CHANGELOG.md')
   .option('-u, --repo-url [url]', 'specify the repo URL for commit links, defaults to checking the package.json');
+
+
+function list (val) {
```

Comment:
This function will have to move above the `module.exports` line due to linting, but when you do, can you make sure that there's only one new line padding the function (as opposed to the two new lines above this function now)? I thought this was in the linter config, but I guess not.